support-frontend/assets/pages/[countryGroupId]/components/thankYouComponent.tsx

```diff
@@ -72,6 +95,8 @@ export type CheckoutComponentProps = {
 	landingPageSettings: LandingPageVariant;
 };
 
+export type ObserverPrint = 'ObserverPaper' | 'ObserverSubscriptionCard';
```

Comment:
this could probably be a enum so we can type check it more strictly.
```suggestion
export enum ObserverPrint {
  Paper:  'ObserverPaper',
  SubscriptionCard: 'ObserverSubscriptionCard
}
```

Then we can check access it like `ObserverPrint.Paper` or even check it like `observerPrint === ObserverPrint. SubscriptionCard`. No more loose string comparison

Reply:
Like this, think good to use enum's more often in codebase, will try to use this approach in future PR's.  Quite a lot of file changes were needed after this but think its there now.  Updated.